Resources/public/css/theme/views/subitemlist.css

```diff
@@ -92,3 +92,77 @@
     left: 50%;
     position: absolute;
 }
+
+.ez-view-subitemlistview .ez-subitem-hovered-priority-cell  .ez-subitem-priority-icons:after {
+    content: "\e606";
+    background: #E6E6E6;
+    color: #737373;
+    border: 1px solid #888;
+}
+
+.ez-view-subitemlistview .ez-subitem-error-priority-cell  .ez-subitem-priority-icons:after {
+    content: "!";
+    background: #BF3E33;
+    color: #fff;
+    border: 1px solid #BF3E33;
+}
+
+.ez-view-subitemlistview .ez-subitem-priority-icons:after {
+    font-weight: bold;
+}
+
+.ez-view-subitemlistview td.ez-subitem-error-priority-cell,
```

Comment:
I think those 2 selectors can be simplified to:
```css
.ez-view-subitemlistview .ez-subitem-error-priority-cell,
.ez-view-subitemlistview tr:nth-child(2n-1) .ez-subitem-error-priority-cell {
```

provided 
```css
.ez-view-subitemlistview .ez-subitemlist-table td {
    background: #fff;
}
 
.ez-view-subitemlistview .ez-subitemlist-table tr:nth-child(2n-1) td {
    background: #fafafa;
}
```
is simplified by removing `.ez-subitemlist-table`.